flowey/flowey_lib_hvlite/src/_jobs/check_clippy.rs

```diff
@@ -152,11 +151,12 @@ impl SimpleFlowNode for Node {
 
                 let mut exclude = vec!["guest_test_uefi".into()];
 
-                // packages containing libfuzzer-sys do not currently build for Aarch64
+                // packages containing libfuzzer-sys do not currently build for (or on) Aarch64
```

Comment:
is this distinction actually relevant?

Reply:
Yes, it doesn't seem like I can cross compile packages with fuzzer for x64 from aarch64

Reply:
I see. maybe tweak the comment/condition a bit to make that support matrix more clear?

e.g: require `matches!(flowey_arch, FlowArch::X86_64)`, rathen than exluding aarch64?

Reply:
here and elsewhere (I see you did the same thing in vmgs_lib)

Reply:
I don't know why that is better since I can't cross compile packages with fuzzer the other way for aarch64 from x64 either so there would still be two conditions, just now with a not

Reply:
maybe I'm not understanding something: do we know how to build these fuzzers / other crates on Aarch64 in _any_ capacity? either natively, or via cross compiling?

if not - shouldn't this just be a single "only build these packages for x86 && on x86?"?

Reply:
`only build these packages for x86 && on x86`
Yes- this is the same as what I have: packages containing libfuzzer-sys do not currently build for (or on) Aarch64. 

We can't build them natively or cross compiled.

Reply:
I understand your point now, I switched it to be x86 only